index.html

```diff
@@ -1,14 +1,459 @@
+                more!
+              </p>
+            </li>
+            <li
+              class="p-3 col-md-2 row m-1 justify-content-center align-items-center"
+            >
+              <img
+                src="./visuals/program_icon_03.png"
+                alt="podium"
+                class="pt-md-2 col-md-auto col-2 program-icon"
+              />
+              <h6
+                class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2"
+              >
+                GDG Forum
+              </h6>
+              <p
+                class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127"
+              >
+                Share your thoughts and opinions with experts for each topic.
+              </p>
+            </li>
+            <li
+              class="p-3 col-md-2 row m-1 justify-content-center align-items-center"
+            >
+              <img
+                src="./visuals/program_icon_04.png"
+                alt="podium"
+                class="pt-md-2 col-md-auto col-2 program-icon"
+              />
+              <h6
+                class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2"
+              >
+                Workshops
+              </h6>
+              <p
+                class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127"
+              >
+                Full-day workshops, hands-on experiences, and debugging events.
+              </p>
+            </li>
+            <li
+              class="p-3 col-md-2 row m-1 justify-content-center align-items-center"
+            >
+              <img
+                src="./visuals/program_icon_05.png"
+                alt="podium"
+                class="pt-md-2 col-md-auto col-2 program-icon"
+              />
+              <h6
+                class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2"
+              >
+                Networking
+              </h6>
+              <p
+                class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127"
+              >
+                Get opportunities to network with Coders around the world.
+              </p>
+            </li>
+          </ul>
+        </section>
+        <footer>
+          <a class="btn-lg text-light" href="#" role="button"
+            >Join devfest 2022</a
+          >
+        </footer>
+      </div>
+    </section>
+    <section id="speakers" class="featured-speakers text-center pt-5 pb-5">
+      <div class="container">
+        <!----- Speakers Header ------->
+        <div class="main-title position-relative">
+          <h2>Featured Speakers</h2>
+        </div>
+        <!----- Speakers Body ------->
+        <ul
+          id="speakers-card"
+          class="row pt-5 pb-5 ps-md-3 pe-md-3 ps-lg-3 pe-lg-3"
+        ></ul>
+      </div>
+    </section>
+    <section id="about">

```

Comment:
As the previous reviewer suggested, it is one of the project requirements to have two separate HTML files. One for the home page and one for the about page. Currently, you only use one HTML file to display both pages' content and use javascript to switch between them. This isn't a very good practice because if you have a website with multiple pages, it will be very difficult to manage all of them by switching with javascript.😥 Please update your project to have two separate HTML files, one for the home page and one for the about page🙏. After doing so, make sure your navbar links link to each page.